contracts/oracles/ZkETHOracle.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { IZkETH } from "../interfaces/IZkETH.sol";
+import { CorrelatedTokenOracle } from "./common/CorrelatedTokenOracle.sol";
+
+/**
+ * @title ZkETHOracle
+ * @author Venus
+ * @notice This oracle fetches the price of zkETH (assuming 1 rzkETH = 1 WETH)
+ */
+contract ZkETHOracle is CorrelatedTokenOracle {
+    /// @notice Constructor for the implementation contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(
+        address zkETH,
+        address weth,
+        address resilientOracle
+    ) CorrelatedTokenOracle(zkETH, weth, resilientOracle) {}
+
+    /**
+     * @notice Gets the amount of WETH for 1 zkETH (assuming 1 rzkETH = 1 WETH)
+     * @return amount Amount of WETH
```

Comment:
nitpick: I think this function returns the amount of rzkETH for 1 zkETH. The assumption is done in the configuration, by setting WETH as the underlying token